### ISSUE TITLE
2.6/3.0 branch and support

### ISSUE THREAD

**Issue by wey-gu** [heyanlong/nebula-php#2]
Thanks!

**Comment by heyanlong** [comment]
An update is expected within two weeks

**Comment by dauvray** [comment]
Hi,
is this repository dead?

**Comment by wey-gu** [comment]
@dauvray welcome to the community!

Are you willing to build things on nebula-php?

**Comment by dauvray** [comment]
yes, I am looking for a package for Laravel like: Laravel MongoDB but I am not sure that this can be applied with graphDatabase.

**Comment by wey-gu** [comment]
Cool, we don't have many php hackers in the NebulaGraph community, thus this repo isn't active, but let me check with authors on pushing NebulaGraph 3.x support for it!

@heyanlong @shyboylpf could we possibly add this support, or help share procedures on how others could help on this?

Thanks!

**Comment by heyanlong** [comment]
@dauvray You need portability Laravel wrapper to load the package

**Comment by dauvray** [comment]
Yes something like this : https://github.com/Vinelab/NeoEloquent

**Comment by wey-gu** [comment]
Are you interested in creating such wrapping project @dauvray based on nebula-php, we would like to help from our side as much as we can :)
